### ISSUE TITLE
How to build and use the proejct

### ISSUE THREAD

**Issue by YORYOR** [google/CallBuilder#7]
Hey,  I find this project occasionally, it's really a good idea.
But the wiki link seems broken, I've no idea how to use it.
Can anyone help to give a user guide doc?

**Comment by matvore** [comment]
Yeah, I had written a doc for how to use it with Eclipse and put it in the wiki, but it's gone now, and I don't know why :(

What IDE are you using? Have you already tried to get it to work?

**Comment by YORYOR** [comment]
I'm using Intellij idea 15, and actually I still don't know how to make it work without a user guide -_-!.
Can u help to recover the wiki,  then I can keep on work that.
Thank you!
